src/htmx.js

```diff
@@ -4686,7 +4686,7 @@ var htmx = (function() {
     if (title) {
       const titleElt = find('title')
       if (titleElt) {
-        titleElt.innerHTML = title
+        titleElt.innerText = title
```

Comment:
The `find` function returns type `Element|null`, but `<title>` is  an `HTMLElement`, so probably just add `/** @type HTMLElement */` between lines 4688 and 4689.

Reply:
Thank you! Fixed! :)